client/src/components/food/FoodItemsSelected/index.js

```diff
@@ -42,7 +42,33 @@ function FoodItemsSelected(props) {
   const [totalCalories, setTotalCalories] = useState(0);
   const [showLoading, setShowLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
+  const [getData, setGetData] = useState(false);
 
+  const handleEditRequest = (userFoodId, foodId) => {
+    history.push(FoodQuantityPath, {
+      categoryId: foodCategoryId,
+      foodId,
+      userFoodId,
+      isEditFood: true,
+    });
+  };
+
+  const handleDeleteRequest = (userFoodRelationId, foodId) => async () => {
+    try {
+      setShowLoading(true);
+      await axios.delete(`/api/v1/category/${foodCategoryId}/food/${foodId}`, {
+        data: { userFoodRelationId },
+      });
+      setErrorMessage('success');
+      setGetData((item) => !item);
```

Comment:
A comment might be helpful, to explain what `getData` is used for.  I'm not sure I understand what's happening (it looks like it starts as a boolean, `false`, on line 45 but then is being set to a function here)